libraries/AP_Common/Location.cpp

```diff
@@ -105,7 +105,7 @@ bool Location::get_alt_cm(AltFrame desired_frame, int32_t &ret_alt_cm) const
 {
 #if CONFIG_HAL_BOARD == HAL_BOARD_SITL
     if (!initialised()) {
-        AP_HAL::panic("Should not be called on invalid location");
+        AP_HAL::panic("Should not be called on invalid location\n Location cannot be (0, 0, 0)");
```

Comment:
I think it's better to have one line than two.
I select the message in GREP for console output.

EX:
![Screenshot from 2021-05-29 12-03-08](https://user-images.githubusercontent.com/646194/120056308-3dc61e00-c076-11eb-9828-01b44ed1fa32.png)

BEFORE
![Screenshot from 2021-05-29 12-02-29](https://user-images.githubusercontent.com/646194/120056312-4a4a7680-c076-11eb-9bac-16dc645b9ba1.png)

Reply:
change to 1 line please, thanks @khancyr !

Reply:
done